copylot/hardware/lights/voltran/test/test_voltran.py

```diff
@@ -0,0 +1,23 @@
+#%%
+from copylot.hardware.lights.voltran import voltran
+
+def demo_get_laser():
+    laser_list = voltran.get_lasers()
+    print(laser_list)
+
+def demo_voltran():
```

Comment:
what does exactly this second demo function doing @edyoshikun ? what is the expected behavior?

Reply:
added more meaningful name `demo_toggle_emission`[04cffcc](https://github.com/czbiohub/coPylot/pull/148/commits/04cffcc53f2925aff1613e78e1b87f70d1fb7497)